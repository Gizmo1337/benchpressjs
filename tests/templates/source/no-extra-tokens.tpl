<!-- IF !isFalse -->
above stuff

  <!-- BEGIN animals -->
    <!-- IF function.isHuman, @index -->
    {animals.name} is human
    <!-- ENDIF function.isHuman -->
  <!-- END animals -->

<!-- ELSE -->
other stuff
<!-- ENDIF !isFalse -->
