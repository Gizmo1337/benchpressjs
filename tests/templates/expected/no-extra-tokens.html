above stuff


  
  
    Human is human

